index.html

```diff
@@ -96,6 +100,145 @@ <h2>Software Developer</h2>
             </a>
         </div>    
         </section>
+<!--Recent work section-->
+        <section class="recent--work--container">
```

Comment:
- [ ] **Please make sure that the work section uses the font and colors (background and font) defined for the template**.

The font and the button color on the website don't match the original design please fix it.

Original | Clone
|:-------------------------:|:-------------------------:|
![image](https://user-images.githubusercontent.com/59168430/120859266-98dda080-c5a1-11eb-9f1d-439989f9b277.png)|![image](https://user-images.githubusercontent.com/59168430/120859212-83687680-c5a1-11eb-9961-fdc55f3facd2.png)